common/spring-cloud-stream-modules-common-configuration/pom.xml

```diff
@@ -29,6 +29,12 @@
 			<optional>true</optional>
 		</dependency>
 		<dependency>
+			<groupId>org.springframework.integration</groupId>
+			<artifactId>spring-integration-sftp</artifactId>
```

Comment:
Why is the SFTP module (and FTP module for that matter) added as part of the common configuration?